### ISSUE TITLE
Improve detection of docker compose version and syntax

### ISSUE THREAD

**Issue by bjoern-arnold** [bcgov/von-network#230]
#171 introduced a function to detect the version of docker compose and -depending on the determined version- sets the correct command syntax. However, the version detection assumes that the command is still available as "docker-compose"
`dockerComposeVersion=$(docker-compose version --short | sed 's~v~~;s~-.*~~')`
This is not the case with my installation (Ubuntu 20.04, Docker version 20.10.16, build aa7e414 with docker-compose plugin v2.5.0).
Since [Compose V2 went GA](https://www.docker.com/blog/announcing-compose-v2-general-availability/) in April I suggest to make the new syntax the default for both the _dockerCompose_ variable and the version detection. As a fallback, the old syntax could still be supported by introducing something like: 
```
dockerComposeVersion=$(docker compose version --short | sed 's~v~~;s~-.*~~')
if [ -z $dockerComposeVersion ]; then
  dockerComposeVersion=$(docker-compose version --short | sed 's~v~~;s~-.*~~')
fi
```

**Comment by WadeBarnes** [comment]
The reason it was not done that way is because the new ` docker compose` command will **always** return the newer docker compose version even if you are not using it and have it disabled in docker-desktop.

Example:
```
$ docker compose version --short
2.6.1

$ docker-compose version --short
1.29.2
```
![image](https://user-images.githubusercontent.com/10504350/180780226-e65b80b8-9b01-4e83-8844-b513f597af1b.png)

---

When you select **Use Docker Compose V2** `docker-compose` will return the new version:

![image](https://user-images.githubusercontent.com/10504350/180780614-aa4bbe20-32f4-4818-a721-6a060c6521d0.png)

```
$ docker-compose version --short
2.6.1

$ docker compose version --short
2.6.1
```

**Comment by WadeBarnes** [comment]
@bjoern-arnold, are you able to switch back and forth between the old and new docker-compose versions in your environment, and if so how does it report the version being used?

**Comment by bjoern-arnold** [comment]
@WadeBarnes no, I'm not abel to switch versions. I assume it's because I've installed Docker Engine only and the new "docker compose" command does not allow switching versions any more (as described e.g. [here](https://stackoverflow.com/questions/70909362/could-not-find-use-docker-compose-v2-check-box-to-disable-v2)). My issue is that there is no "docker-compose" command available any more and thus the manage script fails at line 22

**Comment by WadeBarnes** [comment]
In that case it is more appropriate to detect whether the `docker-compose` command is available on the system first before detecting the version being used.

**Comment by WadeBarnes** [comment]
Would you like to give that a try and submit a PR?

**Comment by bjoern-arnold** [comment]
@WadeBarnes I'm contributing as a Bosch associate and thus I have to get the project approved by our open source officer. This includes an evaluation of the project repo. He found that your policy requires a CONTRIBUTING.md to be present which is not true for this project (see [here](https://github.com/bcgov/BC-Policy-Framework-For-GitHub/blob/master/BC-Gov-Org-HowTo/Cheatsheet.md)). Could you please add this file?

**Comment by WadeBarnes** [comment]
Done
